### PR TITLE
Bumped version and shifted release notes

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

File: docs/release_notes/v0.2.2.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+SMQTK v0.2.2 Release Nodes
+==========================
+This is a minor release. This release primarilly adds classifier algorithm
```
Comment (aashish24) [inline docs/release_notes/v0.2.2.md]: primarilly --> primarily

Comment (Purg) [inline docs/release_notes/v0.2.2.md]: I am literally the worst at spelling.

File: docs/release_notes/v0.2.2.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+SMQTK v0.2.2 Release Nodes
+==========================
+This is a minor release. This release primarily adds classifier algorithm
```
Comment (aashish24) [inline docs/release_notes/v0.2.2.md]: How about:

Version 0.2.2 is a minor release that primarily added classifier algorithm and classification representation support, a new service web application for nearest-neighbors algorithms, and additional documentation.

Also, this release added few more command line tools, especially of note is iqrTrainClassifier.py that can train a classifier based on the saved state of the IQR demonstration application (also a new feature).

**Comment by aashish24** [comment]
Other than minor suggestion, it looks good to me.

**Comment by aashish24** [comment]
LGTM +1

File: docs/release_notes/v0.2.2.md (hunk 2 above)
Comment (Purg) [inline docs/release_notes/v0.2.2.md]: Valid, will integrate.

**Pull request merged**